packages/astro/src/core/client-directive/build.ts

```diff
@@ -0,0 +1,33 @@
+import { build } from 'esbuild';
```

Comment:
nit: we could move the import of `esbuild` inside the function via dynamic import, so we load `esbuild` only when we create client directives.

Reply:
Vite starts up `esbuild` by default so I didn't went with the dynamic import approach to simplify it.